src/rules/order.js

```diff
@@ -17,6 +17,16 @@ function reverse(array) {
   }).reverse()
 }
 
+function alphabetical(firstName, secondName, direction) {
```

Comment:
Name it something like `compareAlphabetically`? Or `isBeforeAlphabetically` (berk). They're ugly names, so if you have better ideas, go for it, but at the moment, without looking at the code and by just looking at places where it's called, it's hard to know what exactly this function does.